data_acquisition/src/calc_center.R

```diff
@@ -35,6 +35,10 @@ calc_center <- function(poly, yaml) {
       # Xiao Yang's code in EE - Yang, Xiao. (2020). Deepest point calculation 
       # for any given polygon using Google Earth Engine JavaScript API 
       # (Version v1). Zenodo. https://doi.org/10.5281/zenodo.4136755
+      # this needs to be in WGS for best results
+      if (yaml$poly_crs != "EPSG:4326" & !is.na(yaml$poly_crs)) {
+        one_wbd <- st_transform(one_wbd, "EPSG:4326")
+      }
```

Comment:
here's that fix that was needed to handle oddball CRS for polygons in order for poi to calculate correctly